internal/services/compute/virtual_machine_scale_set_resource.go

```diff
@@ -1534,6 +1539,9 @@ func flattenAzureRmVirtualMachineScaleSetExtensionProfile(profile *compute.Virtu
 			if properties.AutoUpgradeMinorVersion != nil {
 				e["auto_upgrade_minor_version"] = *properties.AutoUpgradeMinorVersion
 			}
+			if properties.EnableAutomaticUpgrade != nil {
+				e["automatic_upgrade_enabled"] = *properties.EnableAutomaticUpgrade
+			}
```

Comment:
unfortunately this resource is feature-frozen and as such unfortunately we'll need to roll this back:

```suggestion
```